templates/uefi/generic-qemu-uefi-boot-template.jinja2

```diff
@@ -0,0 +1,62 @@
+{% extends 'base/kernel-ci-base.jinja2' %}
```

Comment:
Note: This template is effectively a boot test, because it doesn't have any test cases.  It's OK for now but the results will probably get lost in the noise until boot tests get tidied up.  What we need to have is the `baseline` test plan running on all the platforms, then we can run variants such as these UEFI cases and all the other kinds of boots (ramdisk, nfs...).